config/i18n-tasks.yml

```diff
@@ -170,3 +174,12 @@ ignore_unused:
   - decidim.initiatives.admin.initiatives_settings.update.success
   - decidim.initiatives.admin_log.initiatives_settings.update
   - activemodel.attributes.initiative.area_id
+  - decidim.authorization_handlers.data_authorization_handler.*
+  - decidim.meetings.directory.meetings.index.all
+  - decidim.meetings.meeting.not_allowed
+  - decidim.meetings.meetings.create.*
```

Comment:
Why is there these lines added in a PR that does not concern meetings ?

Reply:
My i18n spec was failing because of these, i thought i could fix it with it